src/app/debug/table/table.component.ts

```diff
@@ -568,7 +625,7 @@ export class TableComponent implements OnInit, OnDestroy {
   }
 
   setUniqueOptions(data: any): void {
-    for (const headerName of this.viewSettings.currentView.metadataNames as string[]) {
+    for (const headerName of this.viewSettings.currentView?.metadataNames as string[]) {
```

Comment:
In the `softFilterList()` method you check if the currentView is not null, why don't you do that here as well.

Reply:
I believe i did it because it was easier and less code than an if statement, but an if statement would be cleaner i think so i changed it to an if statement.